src/BUILD

```diff
@@ -1,29 +1,34 @@
-package(default_visibility=["//visibility:public"])
+package(default_visibility = ["//visibility:public"])
+
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
 
 # Copy the 2.4 version of typescript's typings into the project.
 # We don't want to copy a file that's licensed under an external license into
 # our source tree, and this mechanism allows the :src library to import it.
 genrule(
     name = "typescript-2.4",
-    outs = ["typescript-2.4.d.ts"],
     srcs = ["//test_ts24:node_modules/typescript/lib/typescript.d.ts"],
+    outs = ["typescript-2.4.d.ts"],
     # see https://docs.bazel.build/versions/master/be/general.html#genrule_args
-    cmd = "cp $< $@"
+    cmd = "cp $< $@",
```

Comment:
It's probably worth (for minimizing diff purposes) sumbitting buildifiler CLs separately first.

Reply:
I'm heavily editing this file (adding new transformers, removing obsolete code) - I think it's not worth picking that apart for the 10 lines diff.